src/components/TimeControl/util.ts

```diff
@@ -601,6 +601,30 @@ export function durationString(seconds): string { /* {{{ */
 
     return ret.trim();
 } /* }}} */
+export function daysOnlyDurationString(seconds): string { /* {{{ */
+    let days = Math.floor(seconds / 86400);
+
+    function plurality(num, single, plural): string {
+        num = Math.round(num);
+        if (num > 0) {
+            if (num === 1) {
+                return " " + num + " " + single;
+            }
+            return " " + num + " " + plural;
+        }
+        return "";
+    }
+
+    let ret: string = "";
+    if (days) {
+        ret += plurality(days, _("Day"), _("Days"));
+    }
+    else {
+        ret += _(" 0 Days");
```

Comment:
For translated strings, don't include spaces before or after the string. If you want a space before, do `" " + _("0 Days")`

BUT that's a moot point - this whole case should be dropped, `plurality(days, _("Day"), _("Days"))` is correct when `days === 0` and it saves a translation.

Reply:
sorry not `plurality`, `ngettext`/`npgettext`

Reply:
fixed 😃